### PR TITLE
Add new Debian package support

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

**Comment by jpereira** [comment]
The new CI/RPM build can be seen in https://github.com/FreeRADIUS/pam_radius/runs/4099421813?check_suite_focus=true

File: debian/README.Debian [hunk 1]
```diff
@@ -0,0 +1,18 @@
+NOTE: The Debian version of libpam-radius-auth uses as default configuration
```
Comment (alandekok) [inline debian/README.Debian:1]: We don't need debian-specific notes in our packages.

Just change the default configuration file to `/etc/pam_radius_auth.conf`.

File: debian/README.Debian [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+file /etc/pam_radius_auth.conf.
+Upstream has a default set to /etc/raddb/server that does not fit in Debian.
+Be aware that the documentation references has not been changed and they
+reflect upstream setups.
+
+Since version 2.0.0 this package ships a PAM configuration in
+/usr/share/pam-configs/radius which is turn off as default as an consequence
+of bug report #614967.
```
Comment (alandekok) [inline debian/README.Debian:9]: Yeah, Debian-specific issues don't belong in our repo.  There are distributions using `.deb` which aren't Debian.

File: debian/README.source [hunk 3]
```diff
@@ -0,0 +1,29 @@
+This package is maintained with git-buildpackage(1). It follows DEP-14 for
```
Comment (alandekok) [inline debian/README.source:1]: I don't see why this file is necessary.  Just drop it.  It's all Debian-specific comments.

File: debian/copyright [hunk 4]
```diff
@@ -0,0 +1,51 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: pam_radius
+Source: https://github.com/FreeRADIUS/pam_radius
+
+Files: *
```
Comment (alandekok) [inline debian/copyright:5]: That's not true... the only things from redhat / cryptocard are the `.c` and `.h` files.  Everything else has been completely changed by us.

File: debian/gbp.conf [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Configuration file for git-buildpackage and friends
+
+[DEFAULT]
+compression = gz
+debian-branch = debian/sid
```
Comment (alandekok) [inline debian/gbp.conf:5]: We don't want to have our `.deb` file mention a specific release of Debian.  This file can probably just be dropped.  Debian can do their own release-specific thing.

File: debian/libpam-radius-auth.NEWS [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  The default standard behavior on package installation is to do nothing as the
+  config key 'Default' is set to 'no'. In case you want to use this config file
+  you need to modify at least this option and call 'pam-auth-update' afterwards.
+
+  More information on the PAM configuration manager can be found in
+  PAM-AUTH-UPDATE(8)
+  or online on site:
+  https://manpages.debian.org/unstable/libpam-runtime/pam-auth-update.8.en.html
+
+ -- Carsten Schoenert <c.schoenert@t-online.de>  Thu, 4 Feb 2021 17:51:00 -0100
```
Comment (alandekok) [inline debian/libpam-radius-auth.NEWS:16]: The news text here should be *our* text.  There's no reason to copy the original Debian text.

This makes it look like the package we produce was done by the Debian people, and not us.

**Pull request merged**